pkgs/tools/misc/doitlive/default.nix

```diff
@@ -0,0 +1,25 @@
+{ stdenv, fetchFromGitHub
+, pkgs, python3Packages }:
+
+python3Packages.buildPythonApplication rec {
+  name = "doitlive-${version}";
+  version = "3.0.3";
+
+  src = fetchFromGitHub {
+    owner = "sloria";
+    repo = "doitlive";
+    rev = "${version}";
+    sha256 = "1y1v7f64fhykm7p9yp1ymx1gj6s8qzcq6ygyf3fzwrvw2jjf3n93";
+  };
+
+  propagatedBuildInputs = with python3Packages; [ pkgs.which click ];
```

Comment:
1. `which` should be an argument, not `pkgs`.
2. Is `which` called by doitlive? In that case, I think it's better to patch the sources or add `makeWrapperArgs = ["--prefix PATH : ${which}/bin"];`

Reply:
Removed it